net.crepererum.breitbandmessung.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.crepererum.breitbandmessung</id>
+
+  <name>Breitbandmessung</name>
+  <summary>Allows to measure internet speed.</summary>
+  <url type="homepage">https://breitbandmessung.de/</url>
+
+  <metadata_license>MIT</metadata_license>
+  <project_license>LicenseRef-proprietary=https://breitbandmessung.de/nutzungsbedingungen</project_license>
+
+  <description>
+    <p>
+      Allows to measure internet speed.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">net.crepererum.breitbandmessung.desktop</launchable>
+
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-info">moderate</content_attribute>
+    <content_attribute id="social-location">intense</content_attribute>
+  </content_rating>
+
+  <releases>
+    <release version="3.3.0" date="2022-07-26" />
+  </releases>
```

Comment:
missing `screenshots`

Reply:
I wasn't sure about screenshots and the license, since its proprietary software. But I guess we could count that as "fair use".

Reply:
So I'm unable to find a screenshot of the app hosted on their website. We could deep-link to some newspaper/blog article though, but I'm not sure if this is OK.

Reply:
You can create your own screenshot and upload it either to this git repo or elsewhere.

Reply:
done